test/cpp_extensions/libtorch_agn_2_10_extension/setup.py

```diff
@@ -34,10 +39,12 @@ def run(self):
 def get_extension():
     extra_compile_args = {
         "cxx": [
-            "-fdiagnostics-color=always",
             "-DTORCH_TARGET_VERSION=0x020a000000000000",
```

Comment:
Windows should use `/DTORCH_TARGET_VERSION=0x020a000000000000`?

Reply:
Visual Studio compiler options can be specified using a forward slash (/) or a dash (-).
See https://learn.microsoft.com/en-us/cpp/build/reference/compiler-command-line-syntax?view=msvc-170, for instance.